cobalt/browser/metrics/cobalt_metrics_service_client.cc

```diff
@@ -41,66 +38,38 @@ class CobaltMetricsLogUploader : public metrics::MetricsLogUploader {
                  const std::string& log_hash,
                  const std::string& log_signature,
                  const metrics::ReportingInfo& reporting_info) {
-    // TODO(b/372559349): Add logic here to report the Blob to the WebApp.
-    NOTIMPLEMENTED();
+    LOG(INFO) << "UMA Payload uploading! Hash: " << log_hash;
```

Comment:
I generally prefer DLOG or DVLOG to minimize logspam in the output, but it depends on what you're doing and when you feel you need to see the logs. I suggest changing it but I'll leave it up to you

Reply:
They're all DLOG now